### ISSUE TITLE
Don't warn about oelint.oelint_vars_mispell if the variable is used in the recipe

### ISSUE THREAD

**Issue by priv-kweihmann** [priv-kweihmann/oelint-adv#120]
```
FOO = "a"

do_configure() {
    ./configure ${FOO}
}
``` 
should not generate a warning
